mage_integrations/mage_integrations/destinations/mongodb/target_mongodb/target.py

```diff
@@ -120,3 +121,46 @@ def _process_lines_internal(self, file_input: t.IO[str]) -> t.Counter[str]:
             stats[record_type] += 1
 
         return Counter(**stats)
+
+    def _process_record_message_mongo(self, message_dict: dict) -> None:
+        """Process a RECORD message.
+
+        Args:
+            message_dict: TODO
```

Comment:
nit: what's TODO?

Reply:
This was written on Meltano SDK, just updated on https://github.com/mage-ai/mage-ai/pull/3388/commits/4b1ffcc5db054a47c0e0e9a6df5fcdaf51099e50